### ISSUE TITLE
Scrutinizer complan for a method not existing even when is included via Trait

### ISSUE THREAD

**Issue by javigomez** [scrutinizer-ci/php-analyzer#646]
Hi, I think I found a false positive raised by PHP Analyzer

The scrutinizer compaint:
```
The method taskCheckRoboFileVersion() does not exist on RoboFile. Did you maybe mean checkRoboFileVersion()?
```

Referring to this line:
https://github.com/redCOMPONENT-COM/redCORE/blob/develop/RoboFile.php#L155

This line is calling to a method at a library included via Composer (https://github.com/redCOMPONENT-COM/redCORE/blob/develop/composer.json#L15), the method is located at: https://github.com/redCOMPONENT-COM/robo/blob/master/src/loadTasks.php#L44 and works without issues. So is a false positive at Scrutiziner.

see https://scrutinizer-ci.com/g/redCOMPONENT-COM/redCORE/inspections/01a32eec-d029-4e6b-90ba-e1f124faba80/issues/files/RoboFile.php?status=new&orderField=path&order=asc&issueId=ac85b904-a990-491e-a90b-a650f0260efd

Thanks
Scrutinizer rocks!

**Comment by schmittjoh** [comment]
Was this method maybe only added recently to that library?
